### ISSUE TITLE
viewing networks on map

### ISSUE THREAD

**Issue by kpande** [ceresward/factorio-logiNetChannels#4]
there is currently no way to view networks on the map, i assume this is due to the way the networks are broken out into separate player factions. is there a way to enable my server to show all factions networks?

**Comment by ceresward** [comment]
Yes, unfortunately as far as I know the mod API doesn't allow the channel networks to show on map view.  It's also not possible to see the network in the Logistics GUI (opened by default with the L key).
